force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -126,16 +172,21 @@ public with sharing class ClassesObjectsMethods {
         List<String> scientistNames = new List<String>{'Tim Berners-Lee', 'Alan Turing', 'Grace Hopper', 'Donald Knuth', 'Guido van Rossum', 'Ken Thompson', 'Stephen Hawking'};
 
         // The variable to store the concatenated string
+        String removeNoGNames = '';
 
         // Loop through the list of scientists' names
-
+        for(string scientist : scientistNames){
             // If the name doesn't contain the letter 'G', skip this iteration
-
+            if (scientist.containsIgnoreCase('g')) {
+                removeNoGNames = removeNoGNames + scientist + ',';
```

Comment:
Since you already initialized removeNoGNames with an empty string, you can just use the addition assignment operator here: `removeNoGNames += scientist + ','`;